packages/react-components/src/components/VideoGallery/useVideoTileContextualMenuProps.ts

```diff
@@ -209,25 +301,57 @@ export const useVideoTileContextualMenuProps = (props: {
 
     return { items, styles: {}, calloutProps: { preventDismissOnEvent }, shouldFocusOnContainer: false };
   }, [
-    scalingMode,
-    strings,
-    view,
-    isPinned,
-    onPinParticipant,
-    onUnpinParticipant,
-    onUpdateScalingMode,
+    onMuteParticipant,
+    strings?.muteParticipantMenuItemLabel,
```

Comment:
nit: lets just make this dependant on the strings object changing

Reply:
agree with you, but this was auto added by lint. Keep as it is for now.